pkg/cli/restic.go

```diff
@@ -68,39 +68,48 @@ func (w *ResticWrapper) Backup(resource *api.Restic, fg api.FileGroup) error {
 }
 
 func (w *ResticWrapper) Forget(resource *api.Restic, fg api.FileGroup) error {
+	// Get retentionPolicy for fileGroup, ignore if not found
+	retentionPolicy := api.RetentionPolicy{}
```

Comment:
This check also should happen in the Restic.IsValid() method. 
- There must be a matching RetentionPolicy.